pandas/io/parquet.py

```diff
@@ -180,6 +180,9 @@ def write(
             mode="wb",
             is_dir=partition_cols is not None,
         )
+        if isinstance(path_or_handle, io.BufferedWriter):
+            path_or_handle = path_or_handle.raw.name
```

Comment:
This line seems to be failing mypy tests

`pandas\io\parquet.py:184: error: "RawIOBase" has no attribute "name"  [attr-defined]`

Reply:
Thanks I am opening a new pull request for resolving the mypy issue.